### PR TITLE
Sign up as client

### PR DESCRIPTION
**Pull request by erikwesterberg**
User Story
As a visitor,
In order to become a registered Client,
I would to be able to sign up.
Feature description
https://www.pivotaltracker.com/story/show/168251972

Description of PR
- Creates sign up function with Redux token auth. 
- Installs Redux token auth for authentication.

### CODE REVIEW

File: cypress/integration/visitorCanSignup.spec.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+      "John",
+      "Doe",
+      "johndoe@email.com",
+      "password",
+      "password"
+    );
+    cy.get("#submit-account-button").click();
+    cy.contains("Welcome John!");
+  });
+})
```
Comment (erikwesterberg) [inline cypress/integration/visitorCanSignup.spec.js]: maybe we should have a test for the sad path??

File: src/state/actions/reduxTokenAuthConfig.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    firstName: "first_name",
+    lastName: "last_name",
+    email: "email"
+  },
+  userRegistrationAttributes: {
+    firstName: "first_name",
+    lastName: "last_name",
+    email: "email",
+    password: "password",
+    password_confirmation: "password_confirmation"
```
Comment (pedbr) [inline src/state/actions/reduxTokenAuthConfig.js:16]: we only need to send one password to the API

Comment (erikwesterberg) [reply src/state/actions/reduxTokenAuthConfig.js:16]: fixed

File: cypress/integration/visitorCanSignup.spec.js (hunk 1 above)
Comment (pedbr) [reply cypress/integration/visitorCanSignup.spec.js]: Sad path added

**Pull request merged**